From 0b7f3d1664cf425a314a626e070bab1c1b8e496e Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 23 Jan 2014 13:25:27 +0800
Subject: [PATCH 287/304] ENGR00296231
 com.android.cts.usb.TestUsbTest#testUsbSerial
 failed

Move all USB config update to be init.usb.rc, which aligned with AOSP config

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/etc/init.rc     | 5 -----
 imx6/etc/init.usb.rc | 7 +++++++
 2 files changed, 7 insertions(+), 5 deletions(-)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index 53607d0..1a1b812 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -408,11 +408,6 @@ on boot
 # Enable USB host1'remote wakeup event wakeup android system
     write /sys/bus/platform/drivers/fsl-ehci/fsl-ehci.1/power/wakeup disabled
 
-# serial no, vendor id,etc configure
-    write /sys/class/android_usb/android0/iManufacturer $ro.product.manufacturer
-    write /sys/class/android_usb/android0/iProduct $ro.product.model
-    write /sys/class/android_usb/android0/iSerial $ro.serialno
-
 # Prepare for RIL
     setprop gsm.ril.delay 15
     setprop ro.ril.wake_lock_timeout 300
diff --git a/imx6/etc/init.usb.rc b/imx6/etc/init.usb.rc
index 0d7c29f..a190b66 100644
--- a/imx6/etc/init.usb.rc
+++ b/imx6/etc/init.usb.rc
@@ -1,3 +1,10 @@
+
+on boot
+    write /sys/class/android_usb/android0/iSerial $ro.serialno
+    write /sys/class/android_usb/android0/iManufacturer $ro.product.manufacturer
+    write /sys/class/android_usb/android0/iProduct $ro.product.model
+
+
 # USB rndis configuration
 on property:sys.usb.config=rndis
     write /sys/class/android_usb/android0/enable 0
-- 
1.8.0

